.gitignore

```diff
@@ -24,5 +24,6 @@ private.properties
 .gradle/
 .idea/libraries/
 main/build/
+main/proguard/
```

Comment:
I do this as well, but do not have such a directory...
Would you mind issuing a new PR without this change?
Not that this is highly problematic :) but I first would like to understand why this happens and second I'd like to rather have this separated from the functional change.